utils/utils.go

```diff
@@ -3,13 +3,33 @@ package utils
 import (
 	"fmt"
 	"os"
+	"strconv"
 	"time"
 
 	"github.com/dgraph-io/dgo"
 	"github.com/dgraph-io/dgo/protos/api"
 	"google.golang.org/grpc"
 )
 
+// KafkaConsumerConfig is used to dynamically configurate a new kafka consumer
+type KafkaConsumerConfig struct {
+	GroupID         string
+	RTopic          string
+	WTopic          string
+	IsUserDiscovery bool
+}
+
+// NewKafkaConsumerConfig creates an KafkaConsumerConfig structure based on the
+// provided parameters
+func NewKafkaConsumerConfig(groupID, rTopic, wTopic string, isUserDiscovery bool) *KafkaConsumerConfig {
```

Comment:
I would add an extra, project internal package for Kafka that sets all defaults meaningfully and only exposes the config that we actually change